### PR TITLE
Added SSISDB Catalog Option

### PR DESCRIPTION
**Pull request by maverick126**
Option to pass the SSISDB catalog name.  The catalog connection will default to 'SSISDB' database it not supplied.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The different parameters of the task are explained below:
 * **Authentication**: Select the authentication mode for connecting to the SQL Server. In Windows authentication mode, build service account, is used to connect to the SQL Server. In SQL Server Authentication mode, the SQL login and Password have to be provided in the parameters below.
 * **SQL User name**:  Provide the SQL login to connect to the SQL Server. The option is only available if SQL Server Authentication mode has been selected.
 * **SQL Password**: Provide the Password of the SQL login. The option is only available if SQL Server Authentication mode has been selected.
+* **Initial Catalog**: SSISDB catalog name for the Integration Services.  Default to **SSISDB**
```
Comment (mmajcica) [inline README.md]: Should this be a master, as master was the default value I was passing in previously, in order to maintain compatibility?
https://github.com/mmajcica/DeploySsis/blob/ee5b285f998f7cd050e56f7c12bea520dead63b5/task/ps_modules/ispac.psm1#L9

File: task/task.json [hunk 2]
```diff
@@ -75,6 +75,15 @@
       "visibleRule": "AuthScheme = sqlServerAuthentication",
       "helpMarkDown": "Provide the SQL login to connect to the SQL Server. The option is only available if SQL Server Authentication mode has been selected."
     },
+    {
+      "groupName":"server",
+      "name": "InitialCatalog",
+      "type": "string",
+      "label": "SSISDB Catalog Name",
+      "required": false,
```
Comment (mmajcica) [inline task/task.json]: I think this should be a required param and default it to master, as mentioned above.

**Review by mmajcica — CHANGES_REQUESTED** [review]
Could you please evaluate my comments, eventually push the changes?

Thanks

File: README.md (hunk 1 above)
Comment (maverick126) [reply README.md]: Got it.  It should be 'master' so it will not fail if SSISDB does not exists.

Comment (mmajcica) [reply README.md]: For most of users is master (on-premis).
If you commit the change, then I can merge it and deploy.

**Comment by maverick126** [comment]
InitialCatalog default to master.  Set the InitialCatalog as required.

**Comment by maverick126** [comment]
Made another pull request for the same branch with the suggested changes.

**Comment by mmajcica** [comment]
@maverick126 
Looks good.

Have you actually tested it?

**Comment by Dedac** [comment]
@mmajcica How soon do you think that this could be deployed?

**Comment by mmajcica** [comment]
Tomorrow morning

**Comment by mmajcica** [comment]
@Dedac @maverick126 Done.

Please give it a spin and let me know if any.

**Comment by Dedac** [comment]
@mmajcica It is working perfectly, thanks for getting it out.  It may make sense to add in the docs to set it to SSISDB for SSIS built in with Azure for Data Factory

**Comment by mmajcica** [comment]
Be my guest, I'll be happy to improve this further, so a PR is very
welcome.

**Pull request merged**